pkg/cmd/cli/cmd/importimage.go

```diff
@@ -39,78 +40,241 @@ func NewCmdImportImage(fullName string, f *clientcmd.Factory, out io.Writer) *co
 		Example:    fmt.Sprintf(importImageExample, fullName),
 		SuggestFor: []string{"image"},
 		Run: func(cmd *cobra.Command, args []string) {
-			err := RunImportImage(f, out, cmd, args)
-			kcmdutil.CheckErr(err)
+			if err := opts.Complete(f, args, out); err != nil {
```

Comment:
I see, I must have been looking at older commands before this new pattern. Will change.